src/menu.js

```diff
@@ -0,0 +1,45 @@
+const menu = () => {
+  const menuClasses = 'img-background-menu';
```

Comment:
when the page is loaded (Use document.createElement and document.appendChild
Check my comment....